### ISSUE TITLE
pre-commit with git hooks do not show proper colours in terminal

### ISSUE THREAD

**Issue by thatlittleboy** [pre-commit/pre-commit#2356]
### describe your issue

As per title.

I installed pre-commit via pipx (latest version, 2.18.1), did `pre-commit install`. This is the output of running `git commit -v` in my terminal (kitty, 0.25.0)

<img width="400" src="https://user-images.githubusercontent.com/30731072/164883652-e186ea5d-fd64-46e3-a802-ee4f47aa2ce3.png">

I note that there were several issues opened in the past before reporting similar issues, most notably: https://github.com/pre-commit/pre-commit/issues/1459.

But the issue there seemed to be with Windows, but I'm on Mac OS Monterey 12.2.
A later comment in the same thread mentioned about the same issue with VS Code rendering, but I'm running pre-commit in my terminal (not a file output)

For what it's worth, a few more "facts":
* I'm on `zsh 5.8.1 (x86_64-apple-darwin21.3.0)`
* The same issue happens if I run zsh with no user config, `zsh -f` : there are still no colours in the pre-commit output after running git commit.
* The same issue happens in Terminal.app on my Mac. (again, with plain `zsh -f`) So it doesn't seem related to my terminal of choice (kitty).
* The same issue happens in Terminal.app with plain bash `bash --noprofile --norc`. So it doesn't seem related to the shell.
* I *DO* see colours if I just run `pre-commit run` as per normal. <img width=400 src="https://user-images.githubusercontent.com/30731072/164884115-67b73015-20db-4cb7-b678-d45071c77b37.png">
* I *seem* to recall colours showing up well when I ran `git commit` in the past, let's say 1 or 2 months back. Unless I'm recalling incorrectly, something must've caused this regression but I'm not quite sure what. I tried to rollback to earlier versions of `pre-commit` (2.16.0), but the issue persisted.

### pre-commit --version

pre-commit 2.18.1

### .pre-commit-config.yaml

```yaml
repos:
- repo: https://github.com/pre-commit/pre-commit-hooks
  rev: v4.2.0
  hooks:
  - id: trailing-whitespace
  - id: end-of-file-fixer
  - id: check-yaml
  - id: check-added-large-files
- repo: https://github.com/psf/black
  rev: 22.3.0
  hooks:
  - id: black
    args: [--config, pyproject.toml]
- repo: https://github.com/econchick/interrogate
  rev: 1.5.0
  hooks:
  - id: interrogate
    args: [-c, pyproject.toml]
- repo: https://github.com/terrencepreilly/darglint
  rev: v1.8.1
  hooks:
  - id: darglint
    args: [-v 2]  # this config makes the error messages a bit less cryptic.
- repo: https://github.com/PyCQA/flake8
  rev: 4.0.1
  hooks:
    - id: flake8
      args: [--exclude, nbconvert_config.py]
```

**Comment by asottile** [comment]
this is a regression in git 2.36.0 -- for more info see here: https://twitter.com/pre_commit/status/1516492899306909696

**Comment by ndevenish** [comment]
Should `pre-commit install --color always` be bypassing this? It doesn't seem to change the hook created at all, for me.

**Comment by hugovk** [comment]
This is now fixed in git 2.37.0:

![image](https://user-images.githubusercontent.com/1324225/176890581-1a72ed59-624e-4cc4-b5a3-0d828a517750.png)

**Comment by asottile** [comment]
> Should `pre-commit install --color always` be bypassing this? It doesn't seem to change the hook created at all, for me.

nope, that controls the color of the install command
